backend/spacewalk-backend.changes

```diff
@@ -6,6 +6,7 @@ Wed Jun 10 12:14:57 CEST 2020 - jgonzalez@suse.com
 - supportconfig speedup fixes, add option to not compress spacewalk-debug output dir
 - Prevent failure when syncing from RHEL CDN due extra params (bsc#1171885)
 - Ubuntu repos sync uses http proxy (bsc#1168845)
+- Only check strongest available Ubuntu/Debian repository index checksum
```

Comment:
This entry needs to be added to the top of the file. It got placed in the wrong place probably after last rebase.

Reply:
Oh, I did not know that. I used https://github.com/uyuni-project/uyuni/pull/2278/files as reference and it was added at the bottom in that PR. I'll move it to the top

Reply:
Oh, you man above that entry. Now I got it.

Reply:
As a general rule, you need to add it to the top of the file. In the case of https://github.com/uyuni-project/uyuni/pull/2278/files , that's not true but it's not really a problem because there is no release on top of the new entry.

This is not the case here on this PR, where you're adding the changelog entry below an already release version of this package.

So, general rule, always on top :)